### PR TITLE
Parallel sdc

### PR DESCRIPTION
**Pull request by tlunet**
Last updates on parallel SDC

CI Pipeline : https://gitlab.inria.fr/mschreib/sweet-ci-tests/-/pipelines/774404

Seems that everything went all right :smile:. You can merge this when you want @schreiberx

### CODE REVIEW

File: Makefile
Comment (schreiberx) [inline Makefile:1]: Why did you add this remove statement? This shouldn't be required, is it?
If it is like that, we need to fix it to have this user-based to avoid any clashes.

Comment (tlunet) [reply Makefile:1]: See in SCONS file :

https://github.com/schreiberx/sweet/blob/c236f72ed2dc9710d6b7ecd455990aecfbca2c79/SConstruct#L603-L612

If `USERNAME` is not defined, it just doesn't name the build repository. Happen to me on my system that defines `USER` but not `USERNAME`. To be cross platform, maybe try different environment variables and take the first one that is defined, or use `whoami`, or generate a unique ID ... there is several solutions
